<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1567</width>
    <height>1088</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QToolBox" name="toolBox">
      <property name="minimumSize">
       <size>
        <width>680</width>
        <height>420</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Reference Sans Serif&quot;;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1549</width>
         <height>993</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Business Page</string>
       </attribute>
       <widget class="QGroupBox" name="lookupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>641</width>
          <height>271</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Reference Sans Serif</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(181, 206, 227);</string>
        </property>
        <property name="title">
         <string>Select Location</string>
        </property>
        <widget class="QComboBox" name="stateList">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>191</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>190</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>State</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>30</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>City</string>
         </property>
        </widget>
        <widget class="QListWidget" name="cityList">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>50</y>
           <width>190</width>
           <height>181</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>30</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Zip code</string>
         </property>
        </widget>
        <widget class="QListWidget" name="zipList">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>50</y>
           <width>191</width>
           <height>161</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="businessGroup">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>280</y>
          <width>1421</width>
          <height>651</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>390</width>
          <height>350</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(181, 206, 227);</string>
        </property>
        <property name="title">
         <string>Filter on Categories</string>
        </property>
        <widget class="QTableWidget" name="businessTable">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>60</y>
           <width>911</width>
           <height>231</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
        <widget class="QListWidget" name="categoryList">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>281</width>
           <height>231</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>271</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Select Category (optional)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>30</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Businesses</string>
         </property>
        </widget>
        <widget class="QPushButton" name="searchButton">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>100</y>
           <width>91</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
        <widget class="QPushButton" name="clearButton">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>170</y>
           <width>91</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
</string>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="popularBusiness">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>350</y>
           <width>661</width>
           <height>261</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
        <widget class="QTableWidget" name="successfulBusiness">
         <property name="geometry">
          <rect>
           <x>840</x>
           <y>350</y>
           <width>551</width>
           <height>261</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>320</y>
           <width>361</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Popular Businesses (in zipcode)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>840</x>
           <y>320</y>
           <width>411</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Successful Businesses (in zipcode)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="refreshButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>450</y>
           <width>91</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="zipStatsBox">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>0</y>
          <width>341</width>
          <height>271</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(181, 206, 227);</string>
        </property>
        <property name="title">
         <string>Zipcode Statistics</string>
        </property>
        <widget class="QLabel" name="sumBusLabel">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>50</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="averageStarLabel">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>100</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="reviewCountLabel">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>150</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string># of Businesses</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Average Stars</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>201</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Reference Sans Serif&quot;;</string>
         </property>
         <property name="text">
          <string>Review Count</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="topCategoriesBox">
        <property name="geometry">
         <rect>
          <x>1000</x>
          <y>0</y>
          <width>421</width>
          <height>271</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(181, 206, 227);</string>
        </property>
        <property name="title">
         <string>Top Categories</string>
        </property>
        <widget class="QTableWidget" name="categoryTable">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>31</y>
           <width>381</width>
           <height>221</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1567</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
